responsibleai/responsibleai/_interfaces.py

```diff
@@ -69,6 +69,7 @@ class ErrorAnalysisData:
     tree_features: List[str]
     matrix_features: List[str]
     metric: str
+    importances: list
```

Comment:
do we have to use word "importance" it is sort of confusing.

Reply:
That's a good point, this word is used in the UI directly so that's why it's also used in the backend.  We can change it but 1.) it probably requires some approval and discussion with PM, as well as decision on what new name should be 2.) it seems like it would require a lot of renames that are outside the scope of this PR.  I'll create a task to track this.

Reply:
@xuke444 I created an issue to track this:
https://github.com/microsoft/responsible-ai-widgets/issues/1019